runtime/src/bank.rs

```diff
@@ -679,16 +679,6 @@ impl Bank {
         );
         assert_eq!(bank.epoch_schedule, genesis_config.epoch_schedule);
         assert_eq!(bank.epoch, bank.epoch_schedule.get_epoch(bank.slot));
-        assert_eq!(
```

Comment:
Asserting that the `rent_collector` in a bank loaded from a snapshot is the same as the `rent_collector` in genesis is incorrect.   Rent burn percentage may change